File: src/main/java/flustix/fluxifyed/console/commands/HelpConsoleCommand.java
package flustix.fluxifyed.console.commands;

import flustix.fluxifyed.console.ConsoleCommand;
import flustix.fluxifyed.console.ConsoleCommands;

public class HelpConsoleCommand extends ConsoleCommand {
    public HelpConsoleCommand() {
        super("help", "Shows this help message.");
    }

    @Override
    public void execute(String[] args) {
        ConsoleCommands.LOGGER.info("Available commands:");
        ConsoleCommands.getCommands().forEach((key, command) -> {
            ConsoleCommands.LOGGER.info(" - " + command.name + ": " + command.description);
        });
    }
}

File: Fluxifyed/CC/Commands/HelpConsoleCommand.cs
using Fluxifyed.Logging;

namespace Fluxifyed.CC.Commands; 

public class HelpConsoleCommand : IConsoleCommand {
    public string Name => "help";
    public string Usage => "help";
    public string Description => "Lists all commands";
    
    public void Execute(string[] args) {
        Console.ForegroundColor = ConsoleColor.Yellow;
        Console.Write("Available commands:\n");
        Console.ForegroundColor = ConsoleColor.White;
        
        foreach (var command in ConsoleCommands.Commands) {
            Console.ForegroundColor = ConsoleColor.Yellow;
            Console.Write($"{command.Name} ");
            Console.ForegroundColor = ConsoleColor.White;
            Console.Write($"- {command.Description}\n");
        }
        
        Logger.EmptyLine();
    }
}